examples/react-instantsearch/src/Autocomplete.js

```diff
@@ -0,0 +1,28 @@
+import { autocomplete } from '@algolia/autocomplete-js';
+import React, { createElement, Fragment, useEffect, useRef } from 'react';
+import { render } from 'react-dom';
+
+export function Autocomplete(props) {
+  const containerRef = useRef(null);
+
+  useEffect(() => {
+    if (!containerRef.current) {
+      return undefined;
+    }
+
+    const search = autocomplete({
+      container: containerRef.current,
+      renderer: { createElement, Fragment },
+      render({ children }, root) {
+        render(children, root);
+      },
+      ...props,
+    });
+
+    return () => {
+      search.destroy();
+    };
+  }, []);
```

Comment:
props aren't dynamic now, is that on purpose?

Reply:
Yeah otherwise it re-renders when you click the reset button for instance. I'm not sure if we should react to all prop changes or ignore them all in this example (that would be a source of confusion).

Reply:
ignoring probably fits for most cases until we make a dedicated react version